src/pages/design/icons/library.mdx

```diff
@@ -10,7 +10,7 @@ This is a list of all the icons in the Garden library, independent of context.
 
 import { SvgSearch } from '../../../examples/design/icons/SvgSearch';
 
-<SvgSearch data={props.data.svgIcons} />
+<SvgSearch data={props.data.svgIcons} inputPlaceholder="Filter all icons" />
```

Comment:
This is a search and not a filter correct? I think the content should be "Search icons"